agent_api_rest/src/identity/profiles/mod.rs

```diff
@@ -0,0 +1,78 @@
+use crate::error::IntoApiErrorExt;
+use crate::handlers::{command_handler, query_handler};
+use crate::utils::serde_explicit_null;
+use agent_identity::profile::aggregate::Source;
+use agent_identity::profile::command::ProfileCommand;
+use agent_identity::state::IdentityState;
+use agent_identity::state::{query_profile, PROFILE_ID};
+use agent_shared::config::Logo;
+use axum::{
+    extract::State,
+    response::{IntoResponse, Response},
+    Json,
+};
+use http_api_problem::ApiError;
+use hyper::StatusCode;
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
The endpoint expects a `displayName` for PATCH, but returns a `display_name` on GET. I think we should serialize consistently (using default snake_case).